lib/rails_tracepoint_stack/filter/gem_path.rb

```diff
@@ -0,0 +1,9 @@
+module RailsTracepointStack
```

Comment:
better to isolate it, so we can easily mock this class without affecting `Bundler.load.specs` default behavior